examples/PythonProvider/python-django-webapp-sample/quickstartproject/settings.py

```diff
@@ -0,0 +1,135 @@
+"""
+Django settings for quickstartproject project.
+
+Generated by 'django-admin startproject' using Django 4.0.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.0/ref/settings/
+"""
+
+import os
+from pathlib import Path
+from azure.appconfiguration.provider import load_provider, AzureAppConfigurationKeyVaultOptions
+from azure.identity import DefaultAzureCredential
+
+ENDPOINT = os.environ.get("AZURE_APPCONFIG_ENDPOINT")
+credential = DefaultAzureCredential()
+keyvault_options = AzureAppConfigurationKeyVaultOptions(credential=credential)
+config = load_provider(endpoint=ENDPOINT, key_vault_options=keyvault_options, credential=credential)
```

Comment:
@mrm9084 I found this API name very confusing. Is it possible we can change it to something like `LoadAzureAppConfiguration`? We are not loading a "provider". We are loading data from the service.

Reply:
@zhenlan we are actually "loading the provider". First if you look at the top of the file we are importing `from azure.appconfiguration.provider import load_provider` then the object that is returned from `load_provider` is an instance of `AzureAppConfigurationProvider` which has been loaded with configuration info.

Reply:
This is a bit clearer with IntelliSense where you can see the interaction.